cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -666,9 +680,12 @@ def delete(self):
         """
         Delete this image
         """
-        request = self._provider.gce_compute.images().delete(
-            project=self._provider.project_name, image=self.name)
-        request.execute()
+        (self._provider
```

Comment:
For breaking long lines, using extra parantheses around the expression is preferred to using backslashes, except for "with".